python/cuml/__init__.py

```diff
@@ -70,6 +70,11 @@
 
 from cuml.utils.pointer_utils import device_of_gpu_matrix
 
+# RAFT
+
+from cuml.raft import raft_include_test
```

Comment:
I don't see a `cuml/raft.py` in this PR? Am I missing something here?

Reply:
See https://github.com/rapidsai/raft/blob/branch-0.14/BUILD.md#pythoncython-integration to see how this works. This file is a symlink that gets created by setup.py to the [python code in RAFT](https://github.com/rapidsai/raft/tree/branch-0.14/python), following the rules of the guideline in the build readme of raft I linked above

Reply:
Added the documentation of the functioning the the file `python/README.md` and updated that file to reflect the current Python build system, which was long overdue

Reply:
Makes sense now. Thanks